config/crds/cdap_v1alpha1_cdapmaster.yaml

```diff
@@ -370,12 +370,25 @@ spec:
                 - status
                 type: object
               type: array
+            oldImage:
+              description: Docker image of CDAP backend of the previous iteration.
+              type: string
+            oldUserInterfaceImage:
+              description:  Docker image of CDAP UI of the previous iteration.
+              type: string
+            useOldImage:
```

Comment:
What is this for?

Reply:
Used by the templates to condition on either using oldImage or the image in the spec. We don't want to use the image in the spec until the pre upgrade job is done